site/en/docs/extensions/reference/alarms/index.md

```diff
@@ -2,39 +2,60 @@
 api: alarms
 ---
 
+## Manifest {: #manifest }
+
+To use `chrome.alarms` API, declare the `"alarms"` permission in the [manifest][doc-manifest]:
+
+```json
+{
+  "name": "My extension",
+  ...
+  "permissions": [
+    "alarms"
+  ],
+  ...
+}
+```
+
 ## Examples
 
 The following examples show how to use and respond to an alarm. To try this API,
 install the [Alarm API example][sample-example] from the [chrome-extension-samples][repo-samples]
 repository.
 
-### Set an alarm
+### Set an alarm {: #set-alarm }
 
-The following example sets an alarm in the service worker when the extension is installed, but not when it, Chrome or any shared module is updated.
+The following example sets an alarm in the service worker when the extension is installed, but not when it, Chrome, or any shared module is updated.
 
 {% Label %}service-worker.js:{% endLabel %}
 
-```javascript
-chrome.runtime.onInstalled.addListener(({ reason }) => {
-  if (reason !== chrome.runtime.OnInstalledReason.INSTALL) {
+```js
+chrome.runtime.onInstalled.addListener(async ({ reason }) => {
+  if (reason !== 'install') {
     return;
   }
 
   // Create an alarm so we have something to look at in the demo
-  chrome.alarms.create('demo-default-alarm', {
+  await chrome.alarms.create('demo-default-alarm', {
     delayInMinutes: 1,
     periodInMinutes: 1
   });
 });
 ```
 
-### Respond to an alarm
+{% Aside %}
 
-The following example sets san icon based on the name of the alarm that went off.
+Starting Chrome 117, the number of active alarms is limited to 500. Once this limitation is reached `chrome.alarms.create()` will fail, and [`chrome.runtime.lastError()`][last-error] will be set if using a callback or reject if using promises.
```

Comment:
A few tweaks to hopefully make it read a bit nicer, and removing the brackets from lastError since it's a property.

```suggestion
Starting in Chrome 117, the number of active alarms is limited to 500. Once this limit is reached, `chrome.alarms.create()` will fail. When using a callback, [`chrome.runtime.lastError`][last-error] will be set. When using promises, the promise will be rejected.
```